wiki/History_of_osu!/2022/en.md

```diff
@@ -20,7 +20,7 @@ The third osu! community meeting of the year on 6 February was held concerning v
 
 osu!(lazer) received the Aim Assist mod as a new feature which draws hit objects to the cursor, essentially requiring less cursor movement from the player.[^aim-assist] The mod was renamed to Magnetised one month after the initial implementation.[^aim-assist-rename] Furthermore, the Alternate mod was also added, requiring the player to *alternate* between two keys when hitting circles.[^alternate] The osu!mania game mode in osu!(lazer) got a new Hold Off mod where hold notes get converted to two regular notes.[^hold-off]
 
-Following the first iteration in December last year was another skinning contest, this time with [Tides of Winter](https://osu.ppy.sh/home/news/2021-12-02-skinning-contest-tides-of-winter-announcement) as the theme. The contest concluded with [thetasigma](https://osu.ppy.sh/users/6234482), [BPChingu](https://osu.ppy.sh/users/4236855), and [Redo_](https://osu.ppy.sh/users/7122165) getting third, second, and first place respectively.[^tides-of-winter]
+Following the first iteration in May last year, there was another skinning contest, this time with [Tides of Winter](https://osu.ppy.sh/home/news/2021-12-02-skinning-contest-tides-of-winter-announcement) as the theme. The contest concluded with [thetasigma](https://osu.ppy.sh/users/6234482), [BPChingu](https://osu.ppy.sh/users/4236855), and [Redo_](https://osu.ppy.sh/users/7122165) getting third, second, and first place respectively.[^tides-of-winter]
```

Comment:
If you want to phrase it like this, the sentence may make it seem like the contest concluded in December, which is not true. Tides of Winter actually began in December and lasted until February.

For more information, see: [Skinning contest #1](https://osu.ppy.sh/wiki/de/Contests/Skinning_Contest/1) and [Winter of Tides](https://osu.ppy.sh/wiki/de/Contests/Skinning_Contest/2)